libs/components/datetime/src/lib/modules/datepicker/datepicker-calendar-inner.component.scss

```diff
@@ -58,62 +305,54 @@
   .sky-datepicker-header-right {
     text-align: right;
   }
-}
 
-.sky-theme-modern {
-  .sky-datepicker-calendar-inner {
-    @include mixins.sky-theme-modern-border;
-    border-radius: $sky-theme-modern-border-radius-md;
-    padding: 5px 10px;
-
-    .sky-datepicker-calendar-table-container:focus-visible {
-      .sky-btn-active {
-        @include mixins.sky-theme-modern-border-active;
-      }
-
-      outline: none;
+  table {
+    border-spacing: var(
```

Comment:
Something is going on with the month and year buttons in the table. I see this `border-spacing` here - and I think it is working. But the buttons are collapsing down to their size and are not as wide as the widest in their columns as the design spec specifies.